### PR TITLE
Add notification page heading

### PR DESCRIPTION
**Pull request by sanzeeb3**
Adds a page heading as described in #3

### CODE REVIEW

File: includes/notifications.php [hunk 1]
```diff
@@ -58,15 +58,16 @@ function notification_screen() {
 
 	$output = $unagi_nags;
 
+	$output = '<h2>' . esc_html__( 'Notifications', 'unagi' ) . '</h2>';
```
Comment (mustafauysal) [inline includes/notifications.php]: `h1` would be better

File: includes/notifications.php [hunk 2]
```diff
@@ -58,15 +58,16 @@ function notification_screen() {
 
 	$output = $unagi_nags;
```
Comment (mustafauysal) [inline includes/notifications.php]: this seems to be redundant if we merge this changes

File: includes/notifications.php [hunk 3]
```diff
@@ -58,15 +58,16 @@ function notification_screen() {
...
 		$notification_info = prepare_notification_info();
 
-		$output = $notification_info['content'];
-
+		$output .= $notification_info['content'];
 	}
 
-	if ( empty( $output ) ) {
-		$output = sprintf(
+	if ( empty( $notification_info['content'] ) ) {
```
Comment (mustafauysal) [inline includes/notifications.php:67]: `$notification_info['content']`  can throw a warning when `show_notifications_nicely()` is false

**Review by mustafauysal — CHANGES_REQUESTED** [review]

File: includes/notifications.php (hunk 3 above)
Comment (sanzeeb3) [reply includes/notifications.php:67]: I'm not sure why would that throw a warning, on which PHP?

Comment (mustafauysal) [reply includes/notifications.php:67]: `$notification_info` is in inaccessible scope so `$notification_info['content']` too.

**Comment by mustafauysal** [comment]
@sanzeeb3 I'm not sure why do we need to make this overcomplicated. Can you just add h1 just before `<div id="unagi-notification-center">` ?

File: includes/notifications.php (hunk 3 above)
Comment (sanzeeb3) [reply includes/notifications.php:67]: @mustafauysal I'm not sure about this and the proposal below would solve this anyway, but the docs reads:

> empty() does not generate a warning if the variable does not exist.

**Pull request merged**